### DISCUSSION TITLE
default language config

### DISCUSSION THREAD

**Discussion by zhao-zg** [navidrome/navidrome#2690] [Q&A]
need public config

**Comment by deluan** [comment]
Like [this](https://www.navidrome.org/docs/usage/configuration-options/#:~:text=opus-,DefaultLanguage,-ND_DEFAULTLANGUAGE)? :)

<img width="1246" alt="Screenshot 2023-12-06 at 7 29 37 AM" src="https://github.com/navidrome/navidrome/assets/331353/224a165b-c6c0-4b39-9489-706150b4c0f4">
